src/scripts/seen_posts.css

```
@@ -2,3 +2,7 @@ body:not(.xkit-seen-posts-only-dim-avatar) .xkit-seen-posts-seen:not(:hover),
 body.xkit-seen-posts-only-dim-avatar .xkit-seen-posts-seen:not(:hover) article > :first-child img {
   opacity: 0.5;
 }
+
+body.xkit-seen-posts-hide .xkit-seen-posts-seen {
+  display: none;
+}
```

```suggestion
body.xkit-seen-posts-hide .xkit-seen-posts-seen article {
  display: none;
}
```

Hiding only the article element ensures that k-to-scroll-up still works; without this, if the post above the one you're pressing k while looking at is hidden with this extension, Tumblr's code will be unable to determine its location and will scroll to the top of the page.